src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Admin Page</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+	<nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else{ %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+  </nav>
+
+  <div id="container">
+    <h1>Admin Page</h1>
+
+   <% if (request.getAttribute("unregistered_user") != null) { %>
```

Comment:
I didn't catch this while reading the servlet code, but I think it's a little funny to have the servlet write the message.  I think, in a MVC (Model-view-controller) design, you want to put as much view in the JSP page as possible.  So I would instead do something like:

* Create a LoginState enum in the servlet
* Create a username attribute
* Use those two in the JSP file to decide what to say to the user.